vimeo-networking/src/main/java/com/vimeo/networking/upload/Upload.java

```diff
@@ -0,0 +1,136 @@
+package com.vimeo.networking.upload;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.stag.UseStag;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+
+/**
+ * The representation of the upload object.
+ * <p>
+ * Created by restainoa on 3/27/18.
+ */
+@UseStag
+public class Upload implements Serializable {
+
+    private static final long serialVersionUID = 9209923316093008332L;
+
+    /**
+     * The status of the upload.
+     */
+    public enum Status {
+        @SerializedName("complete")
+        COMPLETE,
+        @SerializedName("error")
+        ERROR,
+        @SerializedName("in_progress")
+        IN_PROGRESS
+    }
```

Comment:
Does status on the Video object still hold this information as well? 🤞  If so, we won't have to do anything.

Reply:
`status` still holds this information, but it is deprecated so I guess we can leave it for now.

Reply:
There is an additional `transcoding` object on the clip representation which is not documented in blueprint from what I can see